apps/frontend/translations/km.json

```diff
@@ -120,14 +120,185 @@
         "NumFeInjure": "Women",
         "NumKidInjure": "Children",
         "NumOldInjure": "Elderly",
-        "NumDisInjure": "Disabled"
+        "NumDisInjure": "Disabled",
+        "NumFamEva": "Families",
+        "NumPeoEva": "People",
+        "NumMeEva": "Men",
+        "NumFeEva": "Women",
+        "NumKidEva": "Children",
+        "NumOldEva": "Elderly",
+        "NumDisEva": "Disabled",
+        "NumFamRe": "Families",
+        "NumPeoRe": "People",
+        "NumMeRe": "Men",
+        "NumFeRe": "Women",
+        "NumKidRe": "Children",
+        "NumOldRe": "Elderly",
+        "NumDisRe": "Disabled",
+        "NumSafePla": "Number of Safe Locations",
+        "NumPeoSEC": "People in Safe Locations",
+        "NumHouAff": "Affected",
+        "NumHouDam": "Damaged",
+        "NumSchoAff": "Affected",
+        "NumSchoDam": "Damaged",
+        "NumAffHeal": "Affected",
+        "NumDamHeal": "Damaged",
+        "NumPagoAff": "Affected",
+        "NumPagoDam": "Damaged",
+        "NumBuilAff": "Affected",
+        "NumBuilDam": "Damaged",
+        "NumShopAff": "Affected",
+        "NumShopDam": "Damaged",
+        "NumWareHAff": "Affected",
+        "NumWareHDam": "Damaged",
+        "NumCraftAff": "Affected",
+        "NumCraftDam": "Damaged",
+        "FarmAff": "Affected",
+        "FarmDam": "Damaged",
+        "SamNabAff": "Affected",
+        "SamNabDam": "Damaged",
+        "PaddyAff": "Affected",
+        "PaddyDam": "Damaged",
+        "CowEva": "Evacuated",
+        "CowDeath": "Dead",
+        "CowMissing": "Missing",
+        "PigEva": "Evacuated",
+        "PigDeath": "Dead",
+        "PigMissing": "Missing",
+        "BirdEva": "Evacuated",
+        "BirdDeath": "Dead",
+        "BirdMissing": "Missing",
+        "RubberRoAff": "Affected",
+        "RubberRoDam": "Damaged",
+        "ConcretAff": "Affected",
+        "ConcretDam": "Damaged",
+        "RuralRoAff": "Affected",
+        "RuralRoDam": "Damaged",
+        "BridgeAff": "Affected",
+        "BridgeDam": "Damaged",
+        "BeleBridAff": "Affected",
+        "BeleBridDam": "Damaged",
+        "DrainageAff": "Affected",
+        "DrainageDam": "Damaged",
+        "DamAff": "Affected",
+        "DamDamaged": "Damaged",
+        "WatGateAff": "Affected",
```

Comment:
Hmmm, it feels very odd to thirty-plicate translations like this. Why don't you just create an `Affected` entry instead and call this as the label in the code?

Reply:
I don't know yet if it is exactly the labels they will want

Reply:
Aren't these the labels that were used before?

Reply:
I added a comment in https://github.com/WFP-VAM/dmp/pull/73